README.md

```diff
@@ -1 +1,12 @@
 # hw01
+
+## My introduction
+
+My name is **Soaham Bharti**. I am a first year PhD student in behavioral marketing at the *Booth School of Business*.
+
+Thing I want to study:
+
+* How consumers and other decision makers navigate uncertainty
+* People's appraisal of algorithms and other forecasting methods
+
+Here is a [link](https://research.chicagobooth.edu/cdr/phd-students/soaham-bharti) to my Booth profile page.
```

Comment:
The link is great, and maybe you could consider inserting a picture the next time. Overall, good job!